infra/modules/providers/azure/cosmosdb/README.md

```diff
@@ -0,0 +1,54 @@
+# Module Azure Cosmos DB
```

Comment:
can you confirm that you rebased from master?

Reply:
Yes, rebased from master this morning.

Reply:
Why are the cosmos module changes marked as additions in this commit?

Reply:
I think we missed the window and need a rebase again... oy

Reply:
Apologies for the delay @iphilpot . Promise I'll review as soon as you post a new commit.